models/Project.model.ts

```diff
@@ -0,0 +1,17 @@
+import { DeletedTask } from "./DeletedTask.model";
+import { Field } from "./Field.model";
+import { Task } from "./Task.model";
+import { User } from "./User.model";
+
+export type Project = {
+  id: string;
+  title: string;
+  creator: User;
+  tasks: Task[];
```

Comment:
it's extra, please add project id field to a task

Reply:
Done